cliTool/index.ts

```diff
@@ -975,7 +976,29 @@ async function app(): Promise<any> {
       return app();
     });
 }
+/**
```

Comment:
why commented out?

Reply:
when there is any issue when running the cli, it's useful to start removing deps and see what specific dependency makes esm resolution fail. 
It's super obscure, and the only way I found to fix this is by trial/error commending and removing some code.